packages/twenty-front/src/modules/object-record/record-picker/multiple-record-picker/states/selectors/multipleRecordPickerPaginationSelectors.ts

```diff
@@ -0,0 +1,66 @@
+import { MultipleRecordPickerComponentInstanceContext } from '@/object-record/record-picker/multiple-record-picker/states/contexts/MultipleRecordPickerComponentInstanceContext';
+import { multipleRecordPickerPaginationState } from '@/object-record/record-picker/multiple-record-picker/states/multipleRecordPickerPaginationState';
+import { createComponentSelectorV2 } from '@/ui/utilities/state/component-state/utils/createComponentSelectorV2';
+
+export const multipleRecordPickerCurrentOffsetSelector =
```

Comment:
I think these selector are not useful
We use selector to limit the re-renders. A selector provides a function on top of an existing state and only trigger re-renders when the result of the function change

Here all these pagination information will likely change together, building a selector does not seems to provide a lot of re-rendering performance improvement